src/bin/cargo/main.rs

```diff
@@ -35,16 +35,43 @@ fn main() {
     }
 }
 
-fn setup_logger() {
-    let env = tracing_subscriber::EnvFilter::from_env("CARGO_LOG");
+fn setup_logger() -> Option<tracing_chrome::FlushGuard> {
+    #![allow(clippy::disallowed_methods)]
+
+    use tracing_subscriber::prelude::*;
 
-    tracing_subscriber::fmt()
+    let env = tracing_subscriber::EnvFilter::from_env("CARGO_LOG");
+    let fmt_layer = tracing_subscriber::fmt::layer()
         .with_timer(tracing_subscriber::fmt::time::Uptime::default())
         .with_ansi(std::io::IsTerminal::is_terminal(&std::io::stderr()))
         .with_writer(std::io::stderr)
-        .with_env_filter(env)
-        .init();
+        .with_filter(env);
+
+    let (profile_layer, profile_guard) =
+        if env_to_bool(std::env::var_os("_CARGO_LOG_PROFILE").as_deref()) {
```

Comment:
While not enforced, there is a convention in this repo of prefixing with double underscores `__CARGO`?